micro_dl/input/tile_stack.py

```diff
@@ -16,46 +15,89 @@ class ImageStackTiler:
     def __init__(self,
                  input_dir,
                  output_dir,
-                 tile_size,
-                 step_size,
+                 tile_dict,
+                 tile_size=[256, 256],
+                 step_size=[64, 64],
+                 depth=1,
                  time_ids=-1,
                  channel_ids=-1,
                  slice_ids=-1,
+                 pos_ids=-1,
                  hist_clip_limits=None,
                  flat_field_dir=None,
-                 isotropic=False):
+                 isotropic=False,
+                 data_format='channels_first'):
         """
```

Comment:
I think we could safely delete the params that would be passed with tile_dict as ImageStackTiler is currently used only in preprocessing_script!